src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -45,8 +45,8 @@ public void testIsFull() {
 
 	@Test
 	public void testPeekaboo() {
-		StackInterface rhymer = new DefaultCountingOutRhymer();
-		final int EMPTY_STACK_VALUE = -1;
+		DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
+		final int EMPTY_STACK_VALUE = 0;

```

Comment:
The original constant should be used (it would be best to use a common constant in a stacks interface).